openlibrary/templates/covers/manage.html

```diff
@@ -58,7 +58,7 @@
 
     <div class="formElement" style="margin:40px 0 0 30px;">
         <button type="submit" value="Save" class="largest">$_("Save")</button>
-        <a class="floaterClose close-popup" href="javascript:;" onclick="parent.\$.fn.colorbox.close();">$_("Cancel")</a>
+        <a class="floaterClose floaterShut--parent close-popup">$_("Cancel")</a>
```

Comment:
`floaterClose` is unused (2refs); can you delete in a future PR? Or add to your WIP as a separate commit.

Reply:
I can remove this in a follow up when I introduce .dialog--close

Reply:
:+1: